src/pages/billing/Project.js

```diff
@@ -221,10 +224,42 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
     _.map('workspace', workspaces)
   ), [billingProject, workspaces])
 
+  const spendReportKey = 'spend report'
+  const maybeLoadTotalCost = reportErrorAndRethrow('Unable to retrieve spend report data')(async () => {
```

Comment:
Currently we get a 404 with no error message if no billing report data is available. I think we will want the billing API to return a more helpful error message in that case.

Note that there are other reasons that one can get a 404 (for example, startDate or endDate are missing), so we can't necessarily assume what a 404 means.